Orange/data/table.py

```diff
@@ -1465,6 +1465,8 @@ def _compute_contingency(self, col_vars=None, row_var=None):
             arr_indi = [e for e, ind in enumerate(col_indi) if f_cond(ind)]
 
             vars = [(e, f_ind(col_indi[e]), col_desc[e]) for e in arr_indi]
+            if vars and arr is self.X and sp.issparse(self.X):
+                self.X = self.X.tocsc()
```

Comment:
I noticed further problems. If there are unknowns, there is also row selection in L1463, which does not work fast with csc.